src/BlocklyInterface/BlocklyEditor.tsx

```diff
@@ -51,12 +63,20 @@ export const BlocklyEditor: FunctionComponent = () => {
         return;
       }
 
-      if (
-        event instanceof Events.BlockMove ||
-        event instanceof Events.BlockChange
-      ) {
+      dispatch(vmSlice.actions.setCode({ code: blocklyRef.current.getCode() }));
+    }
+
+    //function handleBlocklyUiEvent(event: Events.Ui) {
```

Comment:
Any typescript aficionados - the blockly provided types for Events.Ui doesn't for whatever reason contain the additional Events.Ui fields that aren't in the parent class, Events. They are however supported and in [the official docs](https://developers.google.com/blockly/guides/configure/web/events#blocklyeventsui).

Does anyone know if its possible to append to a class type def inside an external module?